src/subcommands/projects.ts

```diff
@@ -101,18 +108,26 @@ async function showProject(args: Args): Promise<void> {
   const api = args.token
     ? API.fromToken(args.token)
     : API.withTokenProvisioner(TokenProvisioner);
-  const [project, domains, pagedBuilds] = await Promise.all([
+  const [project, domains, pagedBuilds, databases, crons] = await Promise.all([
     api.getProject(args.project),
     api.getDomains(args.project),
     api.listDeployments(args.project),
+    api.getProjectDatabases(args.project),
+    api.getDeploymentCrons(args.project, "latest").catch((err) => {
+      // When the project does not have a production deployment, API returns deploymentNotFound
+      if (err instanceof APIError && err.code === "deploymentNotFound") {
+        return null;
+      }
+      throw err;
+    }),
   ]).catch((err) => {
     if (err instanceof APIError && err.code === "projectNotFound") {
       return [null, null, null];
     }
     throw err;
   });
```

Comment:
I guess this needs match the number of promises awaited above, and not it's 5, so this should probably be `return [null, null, null, null, null]`?